admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -183,19 +223,30 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 		 */
 		_toggleHelp: function( e ) {
 			var id = $( this ).attr( 'data-id' );
-
+			var target = $( '.table-help[data-id="' + id + '"]' );
 			e.preventDefault();
 
 			if ( id === undefined ) {
 				return false;
 			}
 
-			$( '.help[data-id="' + id + '"]' ).slideToggle();
+			$( target ).toggleClass( 'show-help' );
+			$( target ).toggleClass( 'hide-help' );
+			$( '.table-help.show-help[data-id="' + id + '"] td p' ).animate(
```

Comment:
WordPress natively has toggle help. Do we need to build it again?

![image](https://user-images.githubusercontent.com/5925563/77682123-c924ba80-6f6c-11ea-8a3d-4693f1256657.png)

Reply:
the existing help functionality didn't work well with the layout used for the setting page. It caused the elements to shift in undesirable ways.